src/display.jl

```diff
@@ -97,7 +97,7 @@ function SyncPlot(
 
         # Draw plot in container
         Plotly.newPlot(
-            gd, $(lowered[:data]), $(lowered[:layout]), $(options)
+            gd, $(lowered), $(options)
```

Comment:
Does this work as expected?

From the plotly.js docs ([ref](https://plotly.com/javascript/plotlyjs-function-reference/#plotlynewplot)) it looks like we can use one of two forms of the `newPlot` function:

```js
Plotly.newPlot(graphDiv, data, layout, config)

Plotly.newPlot(graphDiv, obj)
```

Here I don't believe we are using either